datadog/resource_datadog_security_monitoring_rule.go

```diff
@@ -157,6 +157,25 @@ func datadogSecurityMonitoringRuleSchema() map[string]*schema.Schema {
 			Description: "Queries for selecting logs which are part of the rule.",
 			Elem: &schema.Resource{
 				Schema: map[string]*schema.Schema{
+					"agent_rule": {
+						Type:        schema.TypeList,
+						Optional:    true,
+						Description: "The agent rule.",
+						Elem: &schema.Resource{
+							Schema: map[string]*schema.Schema{
+								"agent_rule_id": {
+									Type:        schema.TypeString,
+									Required:    true,
+									Description: "The Agent rule ID. Must be unique within the rule.",
+								},
+								"expression": {
+									Type:        schema.TypeString,
+									Required:    true,
+									Description: "A Runtime Security expression determines what activity should be collected by the Datadog Agent. These logical expressions can use predefined operators and attributes. Tags cannot be used in Runtime Security expressions. Instead, allow or deny based on tags under the advanced option.",
```

Comment:
No it's not. Indeed we should update the public API (https://github.com/DataDog/datadog-api-spec) to reflect the new naming as there are still some leftovers of the old brand. I will leave this description as it is because I think we should keep description and names consistent with the API documentation, I will create a card so that we can tackle the renaming for all the public clients/providers.